content/_help/verify-your-identity/verify-your-identity-in-person._es.md

```diff
@@ -47,33 +47,12 @@ Si su dirección actual no coincide con la que figura en su documento de identid
 
 Puede traer una copia física o mostrar una copia digital del documento. No puede mostrar una foto o una captura de pantalla del documento. 
 
-## Complete el proceso en la oficina de correos antes la fecha límite
-
-### Ubicaciones disponibles en el área de Washington D.C. 
-
-Puede visitar cualquiera de las siguientes oficinas de correos. No es obligatorio visitar la ubicación que seleccionó en Login.gov.
-
-1. **BALTIMORE — Post Office™**
-   900 E FAYETTE ST RM 118
-   BALTIMORE, MD 21233-9715
-2. **BETHESDA — Post Office™**
-   6900 WISCONSIN AVE STE 100
-   CHEVY CHASE, MD 20815-9996
-3. **FRIENDSHIP — Post Office™**
-   4005 WISCONSIN AVE NW
-   WASHINGTON, DC 20016-9997
-4. **WASHINGTON — Post Office™**
-   900 BRENTWOOD RD NE
-   WASHINGTON, DC 20018-9997
-5. **ARLINGTON — Post Office™**
-   3118 WASHINGTON BLVD
-   ARLINGTON, VA 22201-9998
-6. **ASHBURN — Post Office™**
-   44715 PRENTICE DR
-   DULLES, VA 20101-9996
-7. **MERRIFIELD — Post Office™**
-   8409 LEE HWY
-   MERRIFIELD, VA 22116-9998
+## Complete el proceso en una oficina de correos adherida antes de la fecha límite
+
+### La verificación de identidad en persona está disponible en las oficinas adheridas de Estados Unidos y algunos de sus territorios*. 
+
+Puede buscar una oficina de correos adherida al iniciar el proceso en Login.gov. Una vez que esté listo para verificar su identidad, podrá ir a cualquier oficina de correos donde esté disponible la verificación en persona.
+**\* Territorios de Estados Unidos disponibles:** Islas Vírgenes de Estados Unidos, Samoa Americana, Guam, Puerto Rico y las Islas Marianas del Norte.
```

Comment:
May need an extra new line here, otherwise the text is rendered as a single paragraph with a line break (vs. two paragraphs like in the other languages).

```suggestion

**\* Territorios de Estados Unidos disponibles:** Islas Vírgenes de Estados Unidos, Samoa Americana, Guam, Puerto Rico y las Islas Marianas del Norte.
```